src/Helpers/constants.ts

```diff
@@ -1,7 +1,14 @@
 export const PERMISSIONS = {
   readHosts: 'inventory:hosts:read',
+  useTasks: 'tasks:*:*',
 };
 
 export const INVENTORY_API_ROOT = '/api/inventory/v1';
 export const INVENTORY_HOSTS_ROOT = '/hosts';
 export const INVENTORY_TAGS_ROOT = '/tags';
+
+export const TASKS_API_ROOT = '/api/tasks/v1';
```

Comment:
Same as for api functions, I would continue ith using the prefix, all previous constants start with `INVENTORY_` lets make these `TASKS_`